core/src/main/java/com/linecorp/armeria/client/Endpoint.java

```diff
@@ -123,11 +125,11 @@ public boolean isGroup() {
      * @return the {@link Endpoint} resolved by {@link EndpointGroupRegistry}.
      *         {@code this} if this endpoint is already a host endpoint.
```

Comment:
Does this need to be updated?

Reply:
What do you think, @imasahiro ?

Reply:
Nevermind, it seems like the body of the Javadoc has been updated already.

Reply:
Ah, forgot to add @return section. Updated.